tree/ntuple/v7/inc/ROOT/RPageStorage.hxx

```diff
@@ -782,11 +786,11 @@ public:
    REntryRange GetEntryRange() const { return fEntryRange; }
 
    /// Allocates and fills a page that contains the index-th element. The default implementation searches
-   /// the page and calls LoadPageImpl(). Returns a default-constructed RPage for suppressed columns.
-   virtual RPageRef LoadPage(ColumnHandle_t columnHandle, ROOT::NTupleSize_t globalIndex);
+   /// the page and calls LoadPageImpl(). Returns a default-constructed ROOT::Internal::RPage for suppressed columns.
```

Comment:
FYI, since we don't have any other instances of classes named `RPage`, as far as doxygen is concerned adding the namespace here isn't necessary (it will create the correct link automatically). On the other hand, writing out the full namespace is maybe also not a bad thing. I don't really have an opinion on either, just wanted to point out!